db2-contacts.js

```diff
@@ -4,28 +4,33 @@ const pl = require('pull-level')
 const Plugin = require('ssb-db2/indexes/plugin')
 const isFeed = require('ssb-ref').isFeed
 
-const bValue = Buffer.from('value')
-const bAuthor = Buffer.from('author')
-const bContent = Buffer.from('content')
-const bType = Buffer.from('type')
-const bContact = Buffer.from('contact')
-
-module.exports = function (createLayer) {
+const B_VALUE = Buffer.from('value')
+const B_AUTHOR = Buffer.from('author')
+const B_CONTENT = Buffer.from('content')
+const B_TYPE = Buffer.from('type')
+const B_CONTACT = Buffer.from('contact')
+
+// This index has the following key/values:
+//
+// sourceIdx => { [destIdx1]: edgeValue, [destIdx2]: edgeValue, ... }
+// "feeds" => [feedAIdx, feedBIdx, feedCIdx, ...]
```

Comment:
In ssb-db2 indexes we have these comments to show the shape of the index, so I thought of adding it here too.